### PR TITLE
first steps revamped

### PR DESCRIPTION
**Pull request by ma-zah**

### CODE REVIEW

File: first-steps.mdx [hunk 1]
```diff
@@ -1,137 +1,130 @@
...
-<Frame caption="First page of the setup flow - link to your website, screenshot 09/2023">
+<Frame caption="First page of the setup flow - link to your website, screenshot 07/2024">
   <img src="/images/setup/setup-1-url.png" alt="enter your url screen" />
 </Frame>
 
 ## 2. Name your project
 
-The second page will render a screenshot of the url you provided. It tests that your URL is accessible.
-Now, you need to give your project a name. We'll propose one similar to the url you provided.
+The second page will render a screenshot of the url you provided. We chexk if it is accessible. You can name your project now. We'll propose one similar to the url you provided.
```
Comment (fabiomaienschein) [inline first-steps.mdx:17]:
```suggestion
The second page will render a screenshot of the url you provided. We check if it is accessible. You can name your project now. We'll propose one similar to the url you provided.
```

File: first-steps.mdx [hunk 2]
```diff
@@ -1,137 +1,130 @@
...
     src="/images/setup/setup-2-project.png"
-    alt="name your project screen, screenshot 09/2023"
+    alt="name your project screen, screenshot 07/2024"
   />
 </Frame>
 
 <div class="mt-8" />
 
-## 3. Select test cases to be generated
+## 3. Sign up
```
Comment (fabiomaienschein) [inline first-steps.mdx:29]: Decided to not mention "confirm your email/ set password" flow?

File: first-steps.mdx [hunk 3]
```diff
@@ -1,137 +1,130 @@
...
+    src="/images/setup/setup-5-login-credentials.png"
+    alt="AI agent asks for login test credentials, screenshot 07/2024"
   />
 </Frame>
 
-## 4. Create an account
+## 5. Discover more test cases
 
-Now we need to sign you up. Please, provide an email, so we can get in touch.
+Here comes the really cool part. You can let the AI agent to through your app and discover more tests. Click on `discover`.
```
Comment (fabiomaienschein) [inline first-steps.mdx:69]:
```suggestion
Here comes the really cool part. You can let the AI agent browse through your site to discover and generate more tests. Click on `discover`.
```

File: first-steps.mdx [hunk 4]
```diff
@@ -1,137 +1,130 @@
...
+    src="/images/setup/setup-8-review-steps.png"
+    alt="Test cases that need review, AI discovered and AI auto-generated - screenshot 07/2024 "
   />
 </Frame>
 
-## 6. Approve your test cases
+## 7. Test your website for the first time
 
-Once our AI agent has finished discovery, you can approve or reject each individual draft to generate its steps.
+Once you have active test cases - they are **on**, you can **run** them. Create your first test report. Click on `run all` button and then `run in the app` and see if they found a bug.
```
Comment (fabiomaienschein) [inline first-steps.mdx:111]:
```suggestion
Once you have active test cases - they are **on**, you can **run** them. Create your first test report. Click on `run all` button and then `run in app` and see if they found a bug.
```
Good that you included that info here, test report and run all being connected is sth new users have to learn for conceptually at first I feel like

File: first-steps.mdx [hunk 5]
```diff
@@ -1,137 +1,130 @@
...
+    alt="Run your first test report - screenshot 07/2024 "
     alt="Approve or deny test cases that octomind discovered - screenshot 02/2024 "
   />
 </Frame>
 
-## 7. Publish your test cases
+## 8. See your first test results
 
-Once our AI agent has finished generating steps, you can publish the test case to disable editing and run it in your [Test Reports](/test-reports)
+You have created your first test report, congratulations! If your test is red we will pinpoint you to step in the app that is broken. Find out more about [test reports](/test-reports.mdx) and [debugging your tests](/debugtopus.mdx).
```
Comment (fabiomaienschein) [inline first-steps.mdx:123]: I like the thinking, but I'm not sure if "debugging" is what we want first time users to do/ it's quite technical. Maybe just hint them to "create a new test case" as the next mission to do?

**Review by fabiomaienschein — APPROVED** [review]
thanks for updating, will be busy with opportunity work today still probably but hope to also update some docs soon!

File: first-steps.mdx (hunk 2 above)
Comment (ma-zah) [reply first-steps.mdx:29]: It's kinda implied, I didn't wanna overstretch the steps.

File: first-steps.mdx (hunk 5 above)
Comment (ma-zah) [reply first-steps.mdx:123]: interesting. I'll send a new commit.

**Pull request merged**